packages/react-graphql-universal-provider/src/GraphQLUniversalProvider.tsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
-import {ApolloClient, ApolloClientOptions} from 'apollo-client';
-import {ApolloLink} from 'apollo-link';
-import {InMemoryCache, NormalizedCacheObject} from 'apollo-cache-inmemory';
+import {ApolloClient, ApolloClientOptions, ApolloLink} from '@apollo/client';
+import {InMemoryCache, NormalizedCacheObject} from '@apollo/client/cache';
```

Comment:
It looks like these are available from the root entrypoint, no need to reach deep